src/main/kotlin/no/nav/familie/ba/migrering/services/HentSakTilMigreringService.kt

```diff
@@ -31,24 +31,35 @@ class HentSakTilMigreringService(
             Log.info("Migrering deaktivert, stopper videre jobbing")
             return "Migrering deaktivert, stopper videre jobbing"
         }
-        val personerForMigrering = infotrygdClient.hentPersonerKlareForMigrering(
-            MigreringRequest(
-                page = 4,
-                size = ANTALL_PERSONER_SOM_HENTES_FRA_INFOTRYGD,
-                valg = "OR",
-                undervalg = "OS",
-                maksAntallBarn = 1,
+
+        var antallPersonerMigrert = 0
+        var startSide = 4
+        while (antallPersonerMigrert < antallPersoner) {
+            val personerForMigrering = infotrygdClient.hentPersonerKlareForMigrering(
+                MigreringRequest(
+                    page = startSide,
+                    size = ANTALL_PERSONER_SOM_HENTES_FRA_INFOTRYGD,
+                    valg = "OR",
+                    undervalg = "OS",
+                    maksAntallBarn = 1,
+                )
             )
-        )
+            Log.info("Fant ${personerForMigrering.size} personer for migrering på side $startSide")
+            if (personerForMigrering.isEmpty()) break
 
-        if (personerForMigrering.size > ANTALL_PERSONER_SOM_HENTES_FRA_INFOTRYGD) {
-            Log.error("For manger personer (${personerForMigrering.size}) avbryter migrering")
-            return "For manger personer (${personerForMigrering.size}) avbryter migrering"
+            antallPersonerMigrert = oppretteEllerSkipMigrering(personerForMigrering, antallPersonerMigrert)
+            if (antallPersonerMigrert < antallPersoner) {
+                startSide++
+            }
         }
 
-        Log.info("Fant ${personerForMigrering.size} personer for migrering")
 
-        var antallPersonerMigrert = 0
+
+        return "Migrerte $antallPersoner"
+    }
+
+    private fun oppretteEllerSkipMigrering(personerForMigrering: Set<String>, antallAlleredeMigret: Int): Int {
```

Comment:
Vi bør nok sende med antallPersoner inn, siden den ikke alltid er lik "this.antallPersoner"
```suggestion
    private fun oppretteEllerSkipMigrering(personerForMigrering: Set<String>, antallAlleredeMigret: Int, antallPersonerSomSkalMigreres: Int): Int {
```